www/assets/app.js

```diff
@@ -153,16 +155,28 @@ function addNote (note) {
   var row = document.createElement('tr')
   var amountTd = document.createElement('td')
   var noteTd = document.createElement('td')
+  var deleteTd = document.createElement('td')
 
   amountTd.textContent = note.amount
   noteTd.textContent = note.note
+  deleteTd.innerHTML = '<a href="#" id="delete-item">Delete</a>'
 
+  row.setAttribute('id', note.id)
```

Comment:
one gotcha here: the `id` attribute must not start with a number. `noted.id` is random and sometimes starts with number. The way to workaround this is to prefix the id, with something like 

```js
row.setAttribute('id', 'item- ' + note.id)
```

That way the attribute will look something like this: `id="item-5EA79FA1-8F73-8F8F-AEE8-3861226529A1"` which will always be valid.